litellm/llms/bedrock.py

```diff
@@ -607,6 +621,11 @@ def init_bedrock_client(
     else:
         endpoint_url = f"https://bedrock-runtime.{region_name}.amazonaws.com"
 
+    real_endpoint_url = None
```

Comment:
please can we not have 2 floating variables both with 'endpoint_url' :D 

is it possible for us to have cloudflare logic in 'cloudflare.py' and just have that function wrap this?

having the cloudflare logic in here, looks like it complicates this

Reply:
we could enforce types on functions in here, so any wrapper function can always know what it's going to get